content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -140,7 +142,7 @@ From those buttons, the payment can be registered by clicking :guilabel:`Registe
    :align: center
    :alt: Showcase of the Register Payment button on a confirmed customer invoice.
 
-Doing so reveals a :guilabel:`Register Payment` pop-up form, which is auto-populated with the
+Doing so reveals a :guilabel:`Register Payment` pop-up form, which is autopopulated with the
```

Comment:
```suggestion
Doing so reveals a :guilabel:`Register Payment` pop-up form, which is auto-populated with the
```
While it totally seems like a single word, a hyphen is actually needed in this instance... https://www.dictionary.com/browse/auto-populate